### PR TITLE
Hide/Delete stream item api.

### PR DESCRIPTION
**Pull request by UmeshBhosale**
This api is used for hiding/deleting stream item on news feeds.

### CODE REVIEW

File: easysocial/easysocial/stream.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		}
+		else
+		{
+			$res->success = 0;
+			$res->message = JText::_('PLG_API_EASYSOCIAL_DELETE_NEWSFEED_ITEM_ERROR');
+		}
+		return $res;
+    }
+}
+?>
```
Comment (coolbung) [inline easysocial/easysocial/stream.php]: Closing PHP tag not needed

**Comment by coolbung** [comment]
You need to add a check if the logged in user is the author of stream item that is being hidden or deleted.
Or is this check already being done by the Easysocial code ?

**Comment by coolbung** [comment]
Looks like file has not passed PHPCS

**Comment by UmeshBhosale** [comment]
updated pull request.

**Pull request merged**